backend/src/modules/tournament-tables/core/application/command/AddTournamentTables.ts

```diff
@@ -0,0 +1,11 @@
+import { TableNumber } from '../../domain/TableNumber';
+
+export class AddTournamentTables {
+  readonly tournamentId: string;
+  readonly tablesList: { tableNumber: TableNumber; tableName: string }[];
```

Comment:
komenda nie ma pojęcia (albo nie powinna mieć) o typach domenowych :) chodzi o to, że np użytkownik dodając stół nie ma wiedzy na temat tego jaki typ powinny przyjąć wprowadzane przez niego dane, on sobie patrzy - o tutaj muszę podać numer stołu, więc dla niego to jest zwykły number, dopiero później na etapie obsługi komendy tłumaczymy przychodzące z niej dane na takie typy których oczekuje nasza domain :) (looknij jak coś do command i commandHandlera w module meczy)

Reply:
Tak, wykonujesz command. A potem domena weryfikuje czy odrzuca czy nie. No i typy z domain są "chronione" / "otaczane" przez application. Nikt oprócz application nie powinien o nich wiedzieć (wyjątkiem domain event, w naszej uproszczonej architekturze). Bo wtedy zmiany w domain skutkowałyby zmianami aż do klientów. A domain powinniśmy sobie móc niezależnie modelować.